spec/services/step_nav_publisher_spec.rb

```diff
@@ -29,6 +31,40 @@
         expect(Services.publishing_api).to have_received(:put_content).with(content_id, payload)
       end
     end
+
+    context "marks draft updated" do
+      let(:step_nav) { create(:step_by_step_page_with_steps) }
+
+      it "does not mark submitted for 2i review as draft" do
```

Comment:
I'd rephrase all of these to be less dev-speak:

`it "does not revert to draft if a step by step in `submitted_for_2i` state is edited"`